uint32[] missing_packets
uint32 last_packet_rxd